<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Mohammad Al-Turany/Sergey Linev</author>
 <class>TGo4CreateNewHistogram</class>
 <widget class="QDialog" name="TGo4CreateNewHistogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Histogram</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="histNameLbl">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>histNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="histNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="dataTypeLbl">
       <property name="text">
        <string>Data Type</string>
       </property>
       <property name="buddy">
        <cstring>typeCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeCombo">
       <item>
        <property name="text">
         <string>Float</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dimensionLbl">
       <property name="text">
        <string>Dimension</string>
       </property>
       <property name="buddy">
        <cstring>dimensionCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dimensionCombo">
       <item>
        <property name="text">
         <string>1d</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2d</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox7">
     <property name="title">
      <string>X-Axis</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>7</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel4">
        <property name="text">
         <string>No. of Bins :</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>XNoOfBins</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="XNoOfBins">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="TextLabel5">
        <property name="text">
         <string>Min.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>Xmin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="Xmin">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="TextLabel6">
        <property name="text">
         <string>Max.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>Xmax</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="Xmax">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="YAxisGroup">
     <property name="title">
      <string>Y-Axis</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>7</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel4_2">
        <property name="text">
         <string>No. of Bins :</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>YNoOfBins</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="YNoOfBins">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="TextLabel5_2">
        <property name="text">
         <string>Min.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>Ymin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="Ymin">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="TextLabel6_2">
        <property name="text">
         <string>Max.</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>Ymax</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="Ymax">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="toolTip">
        <string>Create histogram in local workspace.</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>TGo4CreateNewHistogram</receiver>
   <slot>CreateLocalHist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TGo4CreateNewHistogram</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
